src/util/irep_hash_container.h

```diff
@@ -38,34 +39,34 @@ class irep_hash_container_baset
 
   // this is the first level: address of the content
 
-  struct pointer_hash
+  struct pointer_hasht
```

Comment:
In #376 I took the view that those are effectively operations, and thus went for a linter override. I don't claim any authority on that, and your approach seems perfectly valid. Just noting.